src/pages/Leaderboard.tsx

```diff
@@ -1,12 +1,54 @@
 import '../styles.css'
+import React, { useEffect , useContext} from 'react';
+import { LocalizationContext } from '../context/context.tsx';
 
 function Leaderboard() {
 
-    return (
+  const localization = useContext(LocalizationContext)
+
+  const [tableData, setTableData] = React.useState<{ id: number, name: String, guild: String, time: number }[]>([]);
+  useEffect(() => {
+    fetch("https://ylikellotus.lajp.fi")
+      .then((res) => {
+        if (!res.ok) {
+          throw new Error("Failed to fetch leaderboard data");
+        }
+        return res.json();
+      })
+      .then((res) => {
+        console.log("Leaderbopard: ", res);
+        setTableData(res);
+      })
+      .catch((error) => console.error(error));
+  }, []);
+  return (
+    <div className='scroll-container'>
+      <img src='/ScrollTopNew.png' className='scroll-top' />
+      <div className='scroll-center'>
         <div>
-            <p>Leaderboard tähän?</p>
+        {tableData !== null ? (
+          <table>
+            {localization("leaderboardHeaders")}    
+            <tbody>
+              {tableData.slice(0,10000).map((row) => (
+                <tr key={row.id}>
+                  <td>{row.name}</td>
+                  <td>{row.guild}</td>
+                  <td>{row.time}</td>
```

Comment:
As the times are in milliseconds, I think we should convert these to seconds and maybe add the unit, something like
````typescript
`${(row.time/1000.0).toPrecision(4)} s`
````